docs/advanced/document-store/benchmarking.md

```diff
@@ -0,0 +1,122 @@
+# Benchmarking
+## Results
+
+The script `scripts/benchmarking.py` benchmarks DocArray's supported Document Stores in 6 different operations:
+* Create (indexing Documents)
+* Read
+* Update
+* Delete
+* Find Document by vector (Nearest Neighbor Search or Approximate Nearest Neighbor Search depending on the `backend`)
+* Find Document by condition (apply filter)
+
+````{tab} #1: Same HNSW parameters
+
+| Backend       | Create (s) | Read (ms) | Update (ms) | Delete (ms) | Find by vector (s) | Recall at k=10 for | Find by condition (s) |
+|---------------|------------|-----------|-------------|-------------|--------------------|--------------------|-----------------------|
+| Memory        | 0.69       | 0.13      | 0.014       | 0.12        | 1.6                | 1                  | 5.4                   |
```

Comment:
- Number column should be right-align
- Fixed-length, for example, 0.014 vs 44 is not professional. In this case you may want to change unit`ms` to `ns`